gnat2goto/driver/goto_utils.ads

```diff
@@ -13,6 +13,10 @@ package GOTO_Utils is
    function CProver_Void_T return Irep;
    function CProver_Nil_T return Irep;
    function CProver_Bool_T return Irep;
+   function CProver_Nil return Irep;
+
+   --  TODO change this to Irep
+   function Internal_Source_Location return Text_Ptr;
```

Comment:
I guess `Source_Ptr` and `Text_Ptr` are the same?

Reply:
They are, but this should be `Source_Ptr`